### PR TITLE
velodrome: Commit dashboard with first metric

### PR DESCRIPTION
**Pull request by apelisse**
Add dashbaord! The file is generated from grafana

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/592)

### CODE REVIEW

**Comment by apelisse** [comment]
You can also look at it on velodrome.k8s.io, probably easier...

File: velodrome/grafana-stack/dashboards/engineers-velocity.json [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    },
+    {
+      "type": "datasource",
+      "id": "influxdb",
+      "name": "InfluxDB",
+      "version": "1.0.0"
+    }
+  ],
+  "id": null,
+  "title": "Kubernetes Developers Velocity",
```
Comment (spxtr) [inline velodrome/grafana-stack/dashboards/engineers-velocity.json]: nit: "Kubernetes Developer Velocity"

File: velodrome/grafana-stack/dashboards/engineers-velocity.json [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  "hideControls": false,
+  "sharedCrosshair": false,
+  "rows": [
+    {
+      "collapse": false,
+      "editable": true,
+      "height": "250px",
+      "panels": [
+        {
+          "title": "Merge throughput",
```
Comment (spxtr) [inline velodrome/grafana-stack/dashboards/engineers-velocity.json]: I prefer "Merge Rate", but it's not a big deal.

**Comment by spxtr** [comment]
Shiny!

The y axis is "merges per week", and the moving average should say how long it is averaging over. Other than that + nits above it looks awesome.

**Comment by apelisse** [comment]
Updated

**Comment by spxtr** [comment]
Awesome! LGTM

**Pull request merged**